bayesian/api_v1.py

```diff
@@ -1076,6 +1080,61 @@ def get(runtime):
         return response, 200
 
 
+class EmptyBooster(ResourceWithSchema):
+    """Implementation of /empty-booster POST REST API call."""
+
+    method_decorators = [login_required]
+
+    @staticmethod
+    def post():
+        """Handle the POST REST API request."""
+        pom_template = fetch_file_from_github(CORE_DEPENDENCIES_REPO_URL, 'pom.template.xml')
+        remote_repo = request.form.get('gitRepository')
```

Comment:
IIRC we talk about some form of caching there. Well ATM it would be ok as the template is very small, but in the production it won't scale much (let's see)